src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/Formatting/FormattingUtilities.cs

```diff
@@ -53,22 +54,22 @@ public static string AddIndentationToMethod(string method, RazorLSPOptions optio
     ///  The contents of the C# file.
     /// </param>
     /// <returns>The indented method.</returns>
-    public static string AddIndentationToMethod(string method, RazorLSPOptions options, int startAbsoluteIndex, int numCharacterBefore, string source)
+    public static string AddIndentationToMethod(string method, int tabSize, bool insertSpaces, int startAbsoluteIndex, int numCharacterBefore, string source)
```

Comment:
Could you unify the name of this parameter `tabSize/indentSize` across methods?

Reply:
I went with TabSize since this is the option value we are actually using (or at least that's the name of the property in the RazorLSPOptions class), though in general tab size doesn't have to be the indent size (at least in VS). We seem to be treating both as the same though.

Reply:
`tabSize` works for me. Please unify the XML doc comments for those parameters to match then. :smile:

Reply:
Cleaned up the comments